### ISSUE TITLE
Release pymgclient 1.4

### ISSUE THREAD

**Issue by gitbuda** [memgraph/pymgclient#66]
- [ ] Add the latest Python version -> https://github.com/memgraph/pymgclient/pull/67
  - [x] 3.11
  - [x] 3.12
  - [x] 3.13
- [ ] Docs (breaking because older versions will break)
  - [ ] Update https://memgraph.github.io/pymgclient/index.html if needed
  - [ ] Put the breaking note under http://github.com/memgraph/pymgclient/releases

**Comment by gitbuda** [comment]
@katarinasupe, since @mattkjames7 was upgrading GQLAlchemy, it popped into my mind that we should also upgrade the `pymgclient` (we haven't done that for too long time) -> not sure when exactly it's going to happen, but it should in the next few weeks 👀

**Comment by katarinasupe** [comment]
Yep, agreed. I discussed this issue with @mattkjames7 yesterday. Tagging @antejavor for the awareness, if he'll manage to release GQLA while I'm on AL. (also to be aware of the progress here)

**Comment by mattkjames7** [comment]
Done!
